examples/MITRotor_quickstart.ipynb

```diff
@@ -0,0 +1,367 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Geometry"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For the `BEM` model geometry, the user can choose how many points they would like radially and azimuthally. The default, if no geometry is provided to the `BEM` constructor, is 10 points radially and 20 points azimuthally. If the user wants a coarser or finer simulation, they can provide the arguments `Nr` and `Ntheta`. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rotor_geometry = mr.BEMGeometry(Nr = 100, Ntheta = 30)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Tip Loss Model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There are two options for tip loss models, `NoTipLoss` and `PrandtlTipLoss`, which are concrete subtypes of the `TipLossModel` class. As the names suggest, the `NoTipLoss` model does not model tip loss, while the `PrandtlTipLoss` uses Prandtl's tip loss factor. With `PrandtlTipLoss`, there is an optional `root_loss` parameter, which defaults to `True`.\n",
+    "\n",
+    "If no tip loss model is provided to the `BEM`, the default is the `PrandtlTipLoss` with `root_loss` set to true."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "no_tip_loss = mr.NoTipLoss()\n",
+    "prandtl_tip_loss = mr.PrandtlTipLoss(root_loss = True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Momentum Model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There are 5 total momentum models to the user to choose from, all of which are concrete subtypes of the `MomentumModel` class. The options are: `ConstantInduction`, `ClassicalMomentum`, `HeckMomentum`, `UnifiedMomentum`, and `MadsenMomentum`. Each of these different momentum models requires different arguments. These are as follows:\n",
+    "\n",
+    "- `ConstantInduction`: argument `a`, which is the constant induction for the rotor. The default is $\\frac{1}{3}$.\n",
+    "\n",
+    "- `ClassicalMomentum`: no arguments.\n",
+    "\n",
+    "- `HeckMomentum`: arguments `averaging` and `ac`. The options for `averaging` are: `\"sector\"`, `\"annulus\"`, or `\"rotor\"` (with a default of `\"rotor\"`). The default value of `ac`, the initial induction value, is $\\frac{1}{3}$. \n",
+    "\n",
+    "- `UnifiedMomentum`: arguments `averaging` and `beta`. The argument `averaging` has the same options as for the `HeckMomentum` model. The argument `beta` is the shear layer growth rate parameter and has a default value of 0.1403.\n",
+    "\n",
+    "- `MadsenMomentum`: argument `cosine_exponent`, which is an optional exponent on the cosine term in the Madsen model.\n",
```

Comment:
Add reference to this paper for `MadsenMomentum' https://wes.copernicus.org/articles/5/1/2020/